src/infra/main/template.yaml

```diff
@@ -48,19 +48,25 @@ Parameters:
 
 Conditions:
   UsePermissionsBoundary: !Not [ !Equals [ !Ref PermissionsBoundary, 'none' ] ]
+  IsInternal: !Or [ !Equals [!Ref Environment, dev], !Equals [!Ref Environment, build],!Equals [ !Ref Environment, staging]]
 
 Mappings:
   EnvConfig:
     dev:
       LambdaLogLevel: DEBUG
+      AccountDeletionProcessorSNSTopicARN: ""
```

Comment:
If we leave comments next to dev and build to state empty string as pointing to our internal AccountDeletionProcessor SNS topic